### PR TITLE
Improve workspace settings II

### PR DESCRIPTION
**Pull request by holgerkoser**
**What this PR does / why we need it**:
This PR allows debugging of single tests without any change to vscode settings.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user

```

### CODE REVIEW

File: scripts/helper.cjs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  const workspaces = execSync('yarn workspaces list --json')
+    .toString('utf8')
+    .split('\n')
+    .filter(val => val.startsWith('{') && val.endsWith('}') )
+    .map(val => JSON.parse(val)) 
+
+  fs.mkdirSync(path.dirname(filename), { recursive: true })
+  fs.writeFileSync(filename, JSON.stringify(workspaces), 'utf8')
```
[the remark this answers is not in the document]
Comment (petersutter) [reply scripts/helper.cjs:37]: i meant the temporary file `const filename = path.join(os.tmpdir(), 'gardener-dashboard', `workspaces.${hash}.json`)`

**Review by petersutter — APPROVED** [review]
/lgtm

File: scripts/helper.cjs (hunk 1 above)
[the remark this answers is not in the document]
Comment (holgerkoser) [reply scripts/helper.cjs:38]: I have limited the cache validity to the current day.

[the remark this answers is not in the document]
Comment (holgerkoser) [reply scripts/helper.cjs:38]: What do you mean with do we need it as file?

**Pull request merged**